tests/integ/sagemaker/jumpstart/model/test_jumpstart_model.py

```diff
@@ -178,6 +186,40 @@ def test_jumpstart_gated_model(setup):
     assert response is not None
 
 
+def test_jumpstart_gated_model_ic_enabled(setup):
+
+    model_id = "meta-textgeneration-llama-2-7b"
+
+    model = JumpStartModel(
+        model_id=model_id,
+        model_version="3.*",  # version >=3.0.0 stores artifacts in jumpstart-private-cache-* buckets
+        role=get_sm_session().get_caller_identity_arn(),
+        sagemaker_session=get_sm_session(),
+    )
```

Comment:
sanity-check: this test won't work in all regions, is that fine?

Reply:
yes, we have this setup for other tests. if the integ test failures in other regions become an issue, we can add generic support to all JS tests (likely with some annotation)